docs/_posts/examples/SDK Build on Windows OS

```diff
@@ -0,0 +1,148 @@
+Windows OS - Windows 7 Professional (VM) 64bit
+
+NOTE: Windows uses backslash (\) in file path’s; while MAC uses forward slash (/)
+
+List of current install dependencies (find relevant download websites, and install these packages. Use NPM command line install where highlighted):
+
+•	Nodejs6.5
```

Comment:
The SDK is currently tested against node 4.x; it *should* work with 6, but until the test suite is widened, we should stick with 4 here.